[2025-01-13T02:22:59.432+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-13T02:22:59.474+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_py_v4.get_name scheduled__2025-01-12T00:00:00+00:00 [queued]>
[2025-01-13T02:22:59.493+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_py_v4.get_name scheduled__2025-01-12T00:00:00+00:00 [queued]>
[2025-01-13T02:22:59.494+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-01-13T02:22:59.525+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2025-01-12 00:00:00+00:00
[2025-01-13T02:22:59.542+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_py_v4', 'get_name', 'scheduled__2025-01-12T00:00:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/dag_python.py', '--cfg-path', '/tmp/tmp41m5bs33']
[2025-01-13T02:22:59.546+0000] {standard_task_runner.py:105} INFO - Job 108: Subtask get_name
[2025-01-13T02:22:59.550+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4132) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-13T02:22:59.552+0000] {standard_task_runner.py:72} INFO - Started process 4142 to run task
[2025-01-13T02:22:59.653+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_py_v4.get_name scheduled__2025-01-12T00:00:00+00:00 [running]> on host a726eb716cdf
[2025-01-13T02:22:59.809+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khanh' AIRFLOW_CTX_DAG_ID='dag_with_py_v4' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-01-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-12T00:00:00+00:00'
[2025-01-13T02:22:59.811+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-13T02:22:59.842+0000] {python.py:240} INFO - Done. Returned value was: Khanh
[2025-01-13T02:22:59.881+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-13T02:22:59.882+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_py_v4, task_id=get_name, run_id=scheduled__2025-01-12T00:00:00+00:00, execution_date=20250112T000000, start_date=20250113T022259, end_date=20250113T022259
[2025-01-13T02:22:59.934+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-13T02:22:59.977+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-13T02:22:59.981+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
